### ISSUE TITLE
Layer Reordering not working as expected in IE11

### ISSUE THREAD

**Issue by bkutty** [fgpv-vpgf/fgpv-vpgf#1392]
Tested build :http://fgpv.cloudapp.net/demo/users/milesap/build-1.4.0-6/prod/samples/index-fgp-en.html?keys=JOSM,EcoAction,CESI_Other,NPRI_CO,Railways,mb_colour,Airports,Barley,ZoomScale

Layer reordering tests failed in IE11 as the layer could not be placed anywhere in the Legend Panel other than its original position. Accessibility tests done using "reorder" icon also failed. Please investigate

![ie reorder issue](https://cloud.githubusercontent.com/assets/18539315/20405614/f0ac4326-acd7-11e6-8ee1-283a8861b813.png)

**Comment by ChrisLatRNCan** [comment]
IE11 error:
SCRIPT5007: Unable to get property 'item' of undefined or null reference

**Comment by ChrisLatRNCan** [comment]
Problem is related to the use of scope() function in IE11:
https://github.com/angular/angular.js/issues/9515

Possible solution: use Listerine :grimacing:

**Comment by ChrisLatRNCan** [comment]
From app.module.js

`.config(($compileProvider, $mdInkRippleProvider) => {
        // to improve IE performance disable ripple effects globally and debug info
        if (RV.isIE) {
            $mdInkRippleProvider.disableInkRipple();
            $compileProvider.debugInfoEnabled(false);
        }
    });`

It's a bit weird but we need to be in debug mode for IE11 so
`$compileProvider.debugInfoEnabled(true);`

**Comment by AleksueiR** [comment]
What does the angular debug mode do? If it's not impeding performance too much, we can have it on in IE11.

**Comment by ChrisLatRNCan** [comment]
It is definitely slowlyer when the viewer is loading. I've to check a bit more for other operations.

**Comment by milesap** [comment]
Please try removing `$compileProvider.debugInfoEnabled(false);` from `app.module.js` to see if this is a fix.

**Comment by AleksueiR** [comment]
Ops, I didn't realize that trying to improve IE11 perfomance (https://github.com/fgpv-vpgf/fgpv-vpgf/issues/1274) breaks dragula :) Go IE!

**Comment by ChrisLatRNCan** [comment]
@milesap That's what I did for the gif animation.

**Comment by milesap** [comment]
Ok, you're safe to remove it as a fix

**Comment by ChrisLatRNCan** [comment]
1..2..3...removed. That was hard

**Comment by ChrisLatRNCan** [comment]
Thanks for the comments
